result-async/src/main/scala-2.13-/uscala/concurrent/result/AsyncResultImplicits.scala

```diff
@@ -0,0 +1,16 @@
+package uscala.concurrent.result
+
+import scala.collection.generic.CanBuildFrom
+import scala.collection.mutable
+import scala.concurrent.ExecutionContext
+
+trait AsyncResultImplicits {
```

Comment:
Here's the old implementation for Scala <= 2.12. Notice the source directory it's in.